mobly/controllers/android_device_lib/snippet_client.py

```diff
@@ -74,12 +74,26 @@ def stop_app(self):
 
     def _is_app_installed(self):
         """Overrides superclass."""
+        # Check if app is installed
+        try:
```

Comment:
Are all the usages we have currently for checking a grep-like call? (meaning pass if 0 or output non-empty, fail if non-zero or output empty).
If so let's just make a wrapper for that specific case that accepts shell args.